src/remap/mod.rs

```diff
@@ -1,7 +1,5 @@
-pub(crate) mod function;
-
-pub use function::*;
 use lazy_static::lazy_static;
+use remap_functions::*;
```

Comment:
Does this depend on all features in `remap-functions` being enabled?

Reply:
Good point. It does, but this has been updated in https://github.com/timberio/vector/pull/5531, which adds an `all()` method to `remap_functions`, which returns whatever list of functions you've enabled through the feature flags.